lib/ops/monitoring/monitoring.go

```diff
@@ -58,3 +58,23 @@ func GetNamespace(client corev1.CoreV1Interface) (string, error) {
 	}
 	return "", trace.NotFound("service %q was not found", defaults.GrafanaServiceName)
 }
+
+// GetInfluxDBCredentials uses to get username and password for
+// InfluxDB administrator user
+func GetInfluxDBCredentials(client corev1.CoreV1Interface) (username, password []byte, err error) {
+	secret, err := client.Secrets(defaults.MonitoringNamespace).Get(defaults.InfluxDBSecretName, metav1.GetOptions{})
+	if err != nil {
+		return nil, nil, trace.Wrap(rigging.ConvertError(err))
+	}
+
+	username, ok := secret.Data["username"]
+	if !ok {
+		return nil, nil, trace.NotFound("username for InfuxDB administrator not found")
```

Comment:
Not found as in not set? Can you clarify when this error might happen and what could cause it? If the error is user-visible, we might need to add additional context.